addons/rose/addon/styles/rose/components/form/_radio-card.scss

```diff
@@ -0,0 +1,199 @@
+  'hover':
+    "<svg viewBox='0 0 24 24' fill='%236f7682' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z' /></svg>",
+  'focus':
+    "<svg viewBox='0 0 24 24' fill='%231563ff' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z' /></svg>",
+  'checked':
+    "<svg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z' fill='%231563ff'/></svg>",
+  'disabled':
+    "<svg viewBox='0 0 24 24' fill='%23bac1cc' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z' /></svg>",
+  'disabled-checked':
+    "<svg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z' fill='%23bac1cc'/></svg>",
+  'error':
+    "<svg viewBox='0 0 24 24' fill='%23c73445' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z' /></svg>",
+  'error-checked':
+    "<svg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z' fill='%23c73445'/></svg>",
+);
+
+.rose-form-radio-card {
+  &.wide {
+    width:  ($size-m) * 23;
+    .rose-icon {
+      --size: #{$size-m};
+    }
+  }
+
+  &.tile {
+    width: ($size-m) * 15;
+   
+    .rose-icon {
+      --size: #{$size-m + $size-xxs + $size-xxxs};
+    }
+    .rose-form-radio-label {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      height: $size-m * 9;
+      &::after {
+        left:  $size-xl * 3.1;
+      }
+    }
+  }
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .radio-card-overlay {
+    position: absolute;
+    height: $size-l + $size-xxs;
+    background-color: var(--ui-gray-subtler-5);
+    width: 100%;
+    bottom: 0;
+    right: 0;
+  }
+  .rose-form-radio-label {
+    font-style: normal;
+    font-weight: bold;
```

Comment:
I am not sure about this, perhaps @randallmorey can bring more knowledge.

But I am sure I saw in the past in our codebase use `type` to define font size, style and weight at once. I.E, [this code](https://github.com/hashicorp/boundary-ui/blob/main/addons/rose/addon/styles/rose/components/card/_link.scss#L86) `@include type.type(m, semibold);`

Reply:
updated